src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/service/realtime.py

```diff
@@ -1174,7 +1167,7 @@ def realtime_service_run_kubernetes(context, service_name, input_data, verbose):
         if content == "ehostunreach":
             print('Unable to reach the requested host.')
             print('If you just created this service, it may not be available yet. Please try again in a few minutes.')
-        elif '%MatchError' in content:
+        elif '%MatchError' in content or 'No such thing' in content:
```

Comment:
Move print(content) to an else block? Seems redundant to print content if we already print a readable error message.